packages/wonder-blocks-dropdown/components/dropdown-launcher.md

```diff
@@ -0,0 +1,122 @@
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
+const {icons} = require("@khanacademy/wonder-blocks-icon");
+const {StyleSheet} = require("aphrodite");
+
+const dropdownItems = [
+    <ActionItem
+        label="Profile"
+        href="http://khanacademy.org/profile"
+        testId="profile"
+    />,
+    <ActionItem
+        label="Teacher dashboard"
+        href="http://khanacademy.org/coach/dashboard"
+        testId="dashboard"
+    />,
+    <ActionItem
+        label="Settings (onClick)"
+        onClick={() => console.log("user clicked on settings")}
+        testId="settings"
+    />,
+    <ActionItem
+        label="Help"
+        disabled={true}
+        onClick={() => console.log("this item is disabled...")}
+        testId="help"
+    />,
+    <ActionItem
+        label="Feedback"
+        disabled={true}
+        href="/feedback"
+        testId="feedback"
+    />,
+    <SeparatorItem />,
+    <ActionItem
+        label="Log out"
+        href="http://khanacademy.org/logout"
+        testId="logout"
+    />,
+];
+
+<DropdownLauncher
+    disabled={false}
+    menuText="Betsy Appleseed"
+    testId="teacher-menu"
+    menuItems={dropdownItems}
+>
+    {(openDropdown) => (
+        <IconButton
+            icon={icons.caretDown}
+            aria-label="search"
+        />
+    )}
+</DropdownLauncher>
+```
+
+In this example we use a Title component with a DropdownLauncher.
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
+const {icons} = require("@khanacademy/wonder-blocks-icon");
+const {StyleSheet} = require("aphrodite");
+
+const dropdownItems = [
+    <ActionItem
+        label="Profile"
+        href="http://khanacademy.org/profile"
+        testId="profile"
+    />,
+    <ActionItem
+        label="Teacher dashboard"
+        href="http://khanacademy.org/coach/dashboard"
+        testId="dashboard"
+    />,
+    <ActionItem
+        label="Settings (onClick)"
+        onClick={() => console.log("user clicked on settings")}
+        testId="settings"
+    />,
+    <ActionItem
+        label="Help"
+        disabled={true}
+        onClick={() => console.log("this item is disabled...")}
+        testId="help"
+    />,
+    <ActionItem
+        label="Feedback"
+        disabled={true}
+        href="/feedback"
+        testId="feedback"
+    />,
+    <SeparatorItem />,
+    <ActionItem
+        label="Log out"
+        href="http://khanacademy.org/logout"
+        testId="logout"
+    />,
+];
+
+<View>
+    <DropdownLauncher
+        disabled={false}
+        menuText="Betsy Appleseed"
+        testId="teacher-menu"
+        menuItems={dropdownItems}
+    >
+        {(openDropdown) => (
```

Comment:
Once `DropdownLauncher` has been updated to include `state` in the callback params, this should be updated to make use of that `state` to modify the style of the title to show that it works, e.g.
```
{(openDropdown, {focused, hovered, pressed}) => (
     <Title
          aria-role="menu"
          style={[
               focused && styles.focused,
               hovered && styles.hovered,
               pressed && styles.pressed,
          ]}
     >
          Hello, world!
     </Title>
)}
```